spec/errors_spec.js

```diff
@@ -0,0 +1,20 @@
+
+var modal = require('./helpers/modal');
+var prefs = require('./helpers/user_prefs');
+var authoring = require('./helpers/authoring');
+var monitoring = require('./helpers/monitoring');
+
+describe('errors', () => {
+    it('SDESK-3965: render modals with translated specs', () => {
+        prefs.navigateTo();
+        prefs.setLang('English (UK)');
+        prefs.save();
+        monitoring.openMonitoring();
+        monitoring.createFromDeskTemplate();
```

Comment:
Can this be a unit test so it can be run only on the modal and wouldn't depend on monitoring or desk templates?